src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -194,7 +194,7 @@ package object servlets extends DebugEnhancedLogging {
     private def moveIfNotAnArchive(srcItem: FileItem, multiPartLocation: File, targetDir: File, id: UUID): Try[Unit] = {
       val target = targetDir / srcItem.name
       logger.info(s"[$id] staging upload to $target, contentType = ${ srcItem.contentType }")
-      if (srcItem.name.isBlank) Success(()) // skip form field without selected files
+      if (srcItem.name.isEmpty) Success(()) // skip form field without selected files
```

Comment:
Please examine the difference between both methods: `isEmpty` checks for an empty array, `isBlank` checks for an empty string after trimming white space. 
So revert and replace `import  org.scalatra.util.RicherString._` with `import nl.knaw.dans.lib.string._`

However @rvanheest doesn't scalatra's motivation to deprecate also apply to our own library? 

    @deprecated("RicherString#isBlank has been deprecated, it conflicts with Java 11's String#isBlank", "2.7.0")

Might apply to a higher Java version than we are using.

Reply:
@jo-pol since we're still running on Java 8 this isn't really a concern for us right now. It would be once we migrate to a higher version of Java. This is quite unlikely for now, since Scala 2.13 (the newer version that we haven't yet gotten around to upgrading to) still supports Java 8. So let's not bother with this for now and keep the `.isBlank` from our own library.

```suggestion
      if (srcItem.name.isBlank) Success(()) // skip form field without selected files
```